partiql-lang/src/main/kotlin/org/partiql/lang/planner/transforms/plan/RexConverter.kt

```diff
@@ -109,7 +112,7 @@ internal object RexConverter : PartiqlAst.VisitorFold<RexConverter.Ctx>() {
     }
 
     override fun walkExprSessionAttribute(node: PartiqlAst.Expr.SessionAttribute, accumulator: Ctx) = visit(node) {
-        val functionName = when (node.value.text.toUpperCase()) {
+        val functionName = when (node.value.text.uppercase(Locale.getDefault())) {
```

Comment:
Out of curiosity, was this locale change due to a bug?

Reply:
No, `toUpperCase` had warnings since it is deprecated.